Averages/projects/averages_project_3.py

```diff
@@ -3,104 +3,62 @@
 import numpy as np
 
 
-
-#calculate and print out the prediction based on ML
+#calculate and print out the prediction
 def get_prediction(url, data={"A":48,"B":23,"C":38,"D":54}):
     r = requests.post(url, data=json.dumps(data))
     response = getattr(r,'_content').decode("utf-8")
     response = json.loads(response)
     prediction_object = json.loads(response['body'])
-
-    if 'Message' in prediction_object and "dummy response" in prediction_object['Message']:
-        print("Please train your model to get better predictions.")
-    
+    label = ""    
     if 'predicted_label' in prediction_object:
         label = prediction_object['predicted_label']
-    else:
-        label = "This model is unable to predict at this point."
 
+    print("\tLabel: ", label)
+    print("\tModel: ", prediction_object['Model'])
+    #print("\tMessage: ", prediction_object['Message']) #UNCOMMENT WHEN MESSAGE KEY PART OF BODY FOR REGRESSION PROJECTS
     return label
 
 #calculate and print out the prediction based on FORMULA
-def formula(num1=50,num2=70,num3=80,num4=90):
-
-    average = ""
-    num1 = float(num1)
-    num2 = float(num2)
-    num3 = float(num3)
-    num4 = float(num4)
-    average = (num1 + num2 + num3 + num4) / 4
-
-    print("Average via Formula: ", average)
+def formula(num1=50, num2=70, num3=80, num4=90):
 
-    return average
+    label = (float(num1) + float(num2) + float(num3) + float(num4)) / 4
 
-def get_validated_input(question,input_type):
-
-    variable = input(question)
-
-    while True:
-        if input_type == 'float':
-            try:
-                user_input = float(variable)
-            except ValueError:
-                variable = input("You must enter a float (e.g.: 1.3).\n" + question)
-        elif input_type == 'integer':
-            try:
-                user_input = int(variable)
-            except ValueError:
-                variable = input("You must enter an integer (e.g.: 1).\n" + question)
-        elif input_type == 'string':
-             try:
-                user_input = str(variable)
-             except ValueError:
-                variable = input("You must enter a string.\n" + question)
-        break
-    return variable
+    print("Average via Formula")
+    print("\tLabel: ", label)
+    return label
 
 if __name__ == "__main__":
 
```

Comment:
how come this one doesnt have the main function factored out and moved to the top?

Reply:
Ah, thanks for the catch! I must have missed this during updating.